.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - run:
           name: Install clusterctl
           command: |
-            curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.4/clusterctl-linux-amd64 -o clusterctl && chmod +x clusterctl  && sudo mv clusterctl /usr/bin
+            curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.7/clusterctl-linux-amd64 -o clusterctl && chmod +x clusterctl  && sudo mv clusterctl /usr/bin
```

Comment:
cert-manager changed its github organization and that broke clusterctl, so we need a version that fixes that.